src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapperTest.java

```diff
@@ -9,342 +9,383 @@
 
     @Test
     public void testComputeActiveStatusToReady() {
+        // set up
         InstanceType instanceType = InstanceType.COMPUTE;
 
-        InstanceState instanceState1 = OpenStackStateMapper.map(instanceType, "active");
-        InstanceState instanceState2 = OpenStackStateMapper.map(instanceType, "ACTIVE");
+        // exercise
```

Comment:
I think you are swapping, again, actual and expected values ....

Reply:
Fixed!